source/YearlyOverview/YearlyOverview.html

```diff
@@ -11,15 +11,15 @@
         <div id="header">
             <h1 id="index">
                 <a
-                    href="http://127.0.0.1:5500/source/Index/Index.html"
+                    href="../Index/Index.html"
                     style="text-decoration: none"
                     id="back"
                 >
-                    < Index</a
+                    Index</a
```

Comment:
Was the "<" removed on purpose? Stylistically, I liked it more

Reply:
Yeah, the reason I moved it was cause the previous page wasn't actually Index. The appearance contradicts the functionality.